pkg/users/expiration-dialogs.js

```diff
@@ -48,6 +48,7 @@ function AccountExpirationDialogBody({ state, errors, change }) {
                                            invalidFormatText=""
                                            id="account-expiration-input"
                                            value={date}
+                                           appendTo={() => document.body}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17817-20221013-154327-af768310-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)